### PR TITLE
Updated part numbers for as many parts as possible

### PR DESCRIPTION
**Pull request by florianmulatz**
for Mouser, Farnell, Digikey (and the HiLink LD2410 with Aliexpress Link)

### CODE REVIEW

File: kicad/AIOsense/AIOsense.kicad_pcb [hunk 1]
```diff
@@ -12498,10 +12498,4 @@
       f6fe39d4-afed-4ca2-88e0-8e2ceaaf3e99
     )
   )
-  (group "" (id e453a6cb-b766-40db-b5bd-4a4b5cdf469d)
```
Comment (Schluggi) [inline kicad/AIOsense/AIOsense.kicad_pcb:12501]: Is there a reason for the ungrouping? Otherwise I would love to stay with my groups :)

**Review by Schluggi — CHANGES_REQUESTED** [review]
Thank you for the PR. I don't even knew kicad had the option for part numbers.

File: kicad/AIOsense/AIOsense.kicad_pcb (hunk 1 above)
Comment (florianmulatz) [reply kicad/AIOsense/AIOsense.kicad_pcb:12583]: To be honest - I don't know what I did "ungroup" ? I even dond't know about a grouping function in KiCad? Cheers mate

Comment (Schluggi) [reply kicad/AIOsense/AIOsense.kicad_pcb:12583]: So we both learn something new about kicad :)

I can't change your PR, so you have to change it before i can approve.

**Review by florianmulatz — COMMENTED** [review]
Restored Group

**Pull request merged**